src/Registry/registry_instance.jl

```diff
@@ -15,9 +15,8 @@ function to_tar_path_format(file::AbstractString)
 end
 
 # See loading.jl
-const TOML_CACHE = let parser = Base.TOML.Parser()
-    parser.Dates = Dates
-    Base.TOMLCache(parser, Dict{String, Dict{String, Any}}())
+const TOML_CACHE = let parser = TOML.DTParser()
```

Comment:
You are not allowed to parameterize the Base.TOML module with the unrelated code from TOML, as they only share identity by occasional accident as an implementation detail

Reply:
Where do you see `Base.TOML` here?

If you mean `Base.TOMLCache` then we can re-export that via `TOML.Internals.TOMLCache` if need be

Reply:
Base.TOMLCache has a typeassert that you use Base.TOML and not the TOML package